apps/src/code-studio/pd/foorm/editor/components/FoormEditorPreview.jsx

```diff
@@ -0,0 +1,66 @@
+// Preview panel for Foorm editor.
```

Comment:
GitHub didn't pick up that this should be a diff off of apps/src/code-studio/pd/foorm/editor/FoormEditorPreview.jsx, not a brand new component